src/ViewProfile.js

```diff
@@ -1,102 +1,113 @@
 import React from "react";
-import {withRouter, Link} from 'react-router-dom'
-import Header from './Header'
+import "./ViewProfile.css";
+import { withRouter, Link } from "react-router-dom";
+import Header from "./Header";
 
 class ViewProfile extends React.Component {
-    constructor(props) {
-      super(props);
-      this.state = {
-        userId: this.props?.location?.state?.id,
-        name: "",
-        birthday: "",
-        age: 0,
-        bio: "",
-        gender: "Female",
-        genderPreference: "Female",
-        education: "",
-        interests: "",
-        error: "",
-        maxDistance: 10,
-      };
-    }
-    componentDidMount(){
-      const myRequest = new Request('http://127.0.0.1:5000/viewprofile', {
-        method: 'POST',
-        headers: { 'Content-Type': 'application/json' },
-        body: JSON.stringify({"userId": this.state.userId}),
-        });
+  constructor(props) {
+    super(props);
+    this.state = {
+      userId: this.props?.location?.state?.id,
+      name: "",
+      birthday: "",
+      age: 0,
+      bio: "",
+      gender: "Female",
+      genderPreference: "Female",
+      education: "",
+      interests: "",
+      error: "",
+      maxDistance: 10,
+    };
+  }
+  componentDidMount() {
+    const myRequest = new Request("http://127.0.0.1:5000/viewprofile", {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({ userId: this.state.userId }),
+    });
     fetch(myRequest)
-        .then(response => response.json())
-        .then(res => 
-                this.setState({
-                    name: res.name,
-                    age: res.age, 
-                    bio: res.bio,
-                    gender: res.gender,
-                    genderPreference: res.genderPreference,
-                    education: res.education, 
-                    interests: res.interests,
-                    birthday: res.birthday,
-                    maxDistance: res.maxDistance,
-                })
-        ).catch((error) => {
-            console.error(error)
+      .then((response) => response.json())
+      .then((res) =>
+        this.setState({
+          name: res.name,
+          age: res.age,
+          bio: res.bio,
+          gender: res.gender,
+          genderPreference: res.genderPreference,
+          education: res.education,
+          interests: res.interests,
+          birthday: res.birthday,
+          maxDistance: res.maxDistance,
         })
-      }
+      )
+      .catch((error) => {
+        console.error(error);
+      });
+  }
 
-    render() {
-      return (
-        <div className="Profile">
-          <Header id={this.state.userId}/>
-          <form id="profileForm">
-            <h1>View My Profile</h1>
-            <p>Name:</p>
+  render() {
```

Comment:
For this section can you display the results next to each other instead of under?
For example:
Name: Joy 
Birthday: 22-02-1999
etc

Reply:
For all my comments on this please also do the same for edit profile